### PR TITLE
Various changes and fixes

### PR DESCRIPTION
**Pull request by docforcedev**
## High Level Overview of Change

* Fix break on mobile for Join Discord Community so that it doesn't overlap with background
* Add customers link to navbar
* Add homepage header animation
* Revert payid form on homepage
* Update FAQ copy
* Minor React warning fixes

### CODE REVIEW

File: src/components/pages/faq/introduction.tsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
+/* eslint-disable no-irregular-whitespace */
```
Comment (Stormtv) [inline src/components/pages/faq/introduction.tsx:1]: Why was this needed?

File: src/components/pages/faq/introduction.tsx [hunk 2]
```diff
@@ -91,6 +92,98 @@ const Introduction: React.FC = () => {
...
+                ,{' '}
+                <a
+                  href="http://shop.tangem.com/"
+                  className="text-orange-500 focus:underline hover:underline"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  Tangem​{' '}
+                </a>
+                and ​{' '}
```
Comment (Stormtv) [inline src/components/pages/faq/introduction.tsx:137]: two spaces here after and?

File: src/components/pages/faq/introduction.tsx [hunk 3]
```diff
@@ -91,6 +92,98 @@ const Introduction: React.FC = () => {
...
+          description: (
+            <>
+              <p>
+                In the default version of PayID, a receiver trusts their PayID
+                provider to not swap the mapping between PayID to payment
+                address. While this trust is sufficient with custodial wallets,
+                it may not be appropriate in some circumstances, such as with a
+                non-custodial wallet.
+              </p>
+              <br />
```
Comment (Stormtv) [inline src/components/pages/faq/introduction.tsx:167]: Why do we need a br here? I'd recommend mt-4 on the second paragraph

File: src/components/pages/faq/introduction.tsx [hunk 4]
```diff
@@ -128,8 +221,53 @@ const Introduction: React.FC = () => {
...
-            'PayID is designed by devs for devs. You can deploy a PayID server with just a few commands with existing web infrastructure, and then integrate it into an application or account system in just a few lines of code.',
+          description: (
+            <>
+              <p>
+                PayID is designed by devs for devs. You can deploy a PayID
+                server with just a few commands with existing web
+                infrastructure, and then integrate it into an application or
+                account system with just a few lines of code.
+              </p>
+              <br />
```
Comment (Stormtv) [inline src/components/pages/faq/introduction.tsx:232]: Why is BR needed here also?

File: src/components/pages/faq/introduction.tsx [hunk 5]
```diff
@@ -148,6 +286,97 @@ const Introduction: React.FC = () => {
...
+          description: (
+            <>
+              <p>
+                With PayID, you can use human-readable addresses to transmit
+                value in the currency of your choice. The PayID network allows
+                participants to reach one another through a standardized
+                address. For example, alice$wallet.com maps to the corresponding
+                URL https://wallet.com/alice.
+              </p>
+              <br />
```
Comment (Stormtv) [inline src/components/pages/faq/introduction.tsx:323]: Why do we need a br here? I'd recommend mt-4 on the second paragraph

File: src/components/pages/faq/introduction.tsx [hunk 6]
```diff
@@ -148,6 +286,97 @@ const Introduction: React.FC = () => {
...
+          title: 'What are the requirements to run a PayID server?',
+          description: (
+            <>
+              <p>
+                To run a PayID server in a production environment, you should
+                give each component of the stack a vCPU and 500 MB of RAM, and
+                ensure there are at least two High-Availability replicas for
+                each component.
+              </p>
+              <br />
```
Comment (Stormtv) [inline src/components/pages/faq/introduction.tsx:364]: Why do we need a br here? I'd recommend mt-4 on the second paragraph

File: src/components/pages/faq/introduction.tsx [hunk 7]
```diff
@@ -148,6 +286,97 @@ const Introduction: React.FC = () => {
...
+          title: 'What is the PayID reference implementation?',
+          description: (
+            <>
+              <p>
+                The Open Payments Coalition provides a reference implementation
+                of the PayID protocol. Anyone may use this reference
+                implementation or make changes to it. Other implementations can
+                be created, as long as they follow the PayID protocol.
+              </p>
+              <br />
```
Comment (Stormtv) [inline src/components/pages/faq/introduction.tsx:343]: Why do we need a br here? I'd recommend mt-4 on the second paragraph

File: src/components/pages/home/introduction.tsx [hunk 8]
```diff
@@ -10,13 +11,36 @@ import { getDocsLocation } from '../../../utils/config'
...
-  const bgImgStyle = {
-    backgroundImage: `url(${withPrefix('/assets/home/background.svg')})`,
-    backgroundRepeat: 'no-repeat',
-    backgroundPosition: 'center top',
-  }
+  const [backgroundAnimation, setBackgroundAnimation] = useState<
+    AnimationItem
+  >()
+
+  const setBgRef = useCallback((node) => {
```
Comment (Stormtv) [inline src/components/pages/home/introduction.tsx:23]:
```suggestion
  const setBgRef = useCallback((node: HTMLDivElement) => {
```

File: src/components/pages/home/introduction.tsx [hunk 9]
```diff
@@ -10,13 +11,36 @@ import { getDocsLocation } from '../../../utils/config'
...
+    if (backgroundAnimation) {
+      backgroundAnimation.destroy()
+    }
+
+    if (node) {
+      setBackgroundAnimation(
+        lottie.loadAnimation({
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```
Comment (Stormtv) [inline src/components/pages/home/introduction.tsx:31]: If you defined the node's type above you don't see to disable this.

File: src/components/pages/home/introduction.tsx [hunk 10]
```diff
@@ -39,13 +63,24 @@ const Introduction: React.FC = () => {
...
+  const onVideoEnd = useCallback(() => {
+    if (videoRef.current) {
+      videoRef.current.showPreview()
+    }
+  }, [videoRef])
+
   return (
     <Wave
-      style={bgImgStyle}
+      style={{ zIndex: 1 }}
```
Comment (Stormtv) [inline src/components/pages/home/introduction.tsx:74]: Could we just use z-10 here?

File: src/components/pages/home/introduction.tsx [hunk 11]
```diff
@@ -39,13 +63,24 @@ const Introduction: React.FC = () => {
...
       wave="white"
       spacing="sm"
       waveClassNames="bg-contain lg:bg-auto xl:bg-contain"
     >
+      <div
+        className="absolute left-0 right-0 lottie-player"
+        ref={setBgRef}
+        style={{ zIndex: '-1' }}
```
Comment (Stormtv) [inline src/components/pages/home/introduction.tsx:82]: Could we extend the z-index in tailwind to include a negative index?

https://tailwindcss.com/docs/z-index#negative-values

**Review by Stormtv — CHANGES_REQUESTED** [review]

File: src/components/pages/faq/introduction.tsx (hunk 1 above)
Comment (docforcedev) [reply src/components/pages/faq/introduction.tsx:1]: Something weird was happening with copy paste and I forgot to remove it. Fixed.

File: src/components/pages/faq/introduction.tsx (hunk 2 above)
Comment (docforcedev) [reply src/components/pages/faq/introduction.tsx:137]: Mistake during refactor, fixed

File: src/components/pages/faq/introduction.tsx (hunk 4 above)
Comment (docforcedev) [reply src/components/pages/faq/introduction.tsx:232]: It creates a paragraph break in the text, without it the 2 paragraphs run together.

Comment (docforcedev) [reply src/components/pages/faq/introduction.tsx:232]: But I see your point, br shouldn't be used for presentation. Just bad habit.

**Review by Stormtv — APPROVED** [review]
Looks good thanks for making those changes
